pkg/cmd/publish/publish.go

```diff
@@ -185,16 +186,42 @@ func (cmd *publishCmd) run(f *cmdutil.Factory, info *publishInfo, options *contr
 		}
 	}
 
-	fmt.Fprintf(cmd.f.IOStreams.Out, "\nYour Domain name: %s\n", domain.GetDomainName())
+	var domainReturnedName []string
+	domainReturnedName = append(domainReturnedName, domain.GetDomainName())
```

Comment:
A more direct way to this do this is to initialize the slice directly
```suggestion
	domainReturnedName := []string{domain.GetDomainName()}
```

Reply:
done